otx/algorithms/action/utils/convert_public_data_to_cvat.py

```diff
@@ -0,0 +1,260 @@
+    name = etree.Element("name")
+    name.text = f"v{i}"
+    task.append(name)
+
+    size = etree.Element("size")
+    size.text = str(n_frames)  ####
+    task.append(size)
+
+    mode = etree.Element("mode")
+    mode.text = "interpolation"
+    task.append(mode)
+
+    overlap = etree.Element("overlap")
+    overlap.text = "2"  ####
+    task.append(overlap)
+
+    bugtracker = etree.Element("bugtracker")
+    bugtracker.text = ""
+    task.append(bugtracker)
+
+    created = etree.Element("created")
+    created.text = ""
+    task.append(created)
+
+    updated = etree.Element("updated")
+    updated.text = ""
+    task.append(updated)
+
+    start_frame = etree.Element("start_frame")
+    start_frame.text = "0"
+    task.append(start_frame)
+
+    stop_frame = etree.Element("stop_frame")
+    stop_frame.text = str(n_frames)
+    task.append(stop_frame)
+
+    frame_filter = etree.Element("frame_filter")
+    frame_filter.text = "1"
+    task.append(frame_filter)
+
+    z_order = etree.Element("z_order")
+    z_order.text = str(True)
+    task.append(z_order)
+
+    labels = etree.Element("labels")
+    task.append(labels)
+
+    label = etree.Element("label")
+    labels.append(label)
+
+    name = etree.Element("name")
+    name.text = str(class_idx)
+    label.append(name)
+
+    attributes = etree.Element("attributes")
+    attributes.text = ""
+    label.append(attributes)
+
+    segments = etree.Element("segments")
+    segments.text = ""
+    task.append(segments)
+
+    original_size = etree.Element("original_size")
+    task.append(original_size)
+
+    sample_frame = cv2.imread(osp.join(video_path, frame_list[0]))
+    original_size_width = etree.Element("width")
+    original_size_width.text = str(sample_frame.shape[1])
+    original_size.append(original_size_width)
+
+    original_size_height = etree.Element("height")
+    original_size_height.text = str(sample_frame.shape[0])
+    original_size.append(original_size_height)
+
+    return annotations, sample_frame.shape
+
+
+# classification
+def convert_jester_dataset_to_datumaro(src_path, dst_path):
+    """Convert Jester dataset to multi-video CVAT (Datumaro) format."""
+    ## Prepare dst_path
+    frames_dir_path = osp.join(src_path, "frames")
+
+    phases = ["train", "val", "test"]
+    for phase in phases:
+        txt_path = osp.join(src_path, f"{phase}_list_rawframes.txt")
+        with open(txt_path, "r", encoding="utf-8") as txt:
+            pathlib.Path(osp.join(dst_path, phase)).mkdir(parents=True, exist_ok=True)
+
+            for i, line in enumerate(txt.readlines()):
+                video_dir, _, class_idx = line[:-1].split(" ")
+
+                video_path = osp.join(frames_dir_path, video_dir)
+                frame_list = os.listdir(video_path)
+
+                shutil.copytree(
+                    video_path,
+                    osp.join(
+                        dst_path,
+                        phase,
+                        f"video_{i}/images",
```

Comment:
`images` could be as standalone argument of join to avoid hardcoded path separator usage or os.path.sep can be used instead of `/` or + operation from pathlib.